product_search/main.py

```diff
@@ -0,0 +1,53 @@
+"""Return the inventory details for an item in the product inventory"""
+
+import sys
+from product_search import (
+    validate_item,
+    inventory_details_for_user_item,
+    _style_dictionary_items_for_output,
+    return_product_details,
+)
+from data_management import access_json_data
+
+from user_input import get_item_name_from_user
+
+
+def main(filename):
+    """
+    Calls to functions to get valid user input, 
+    retrieve inventory details based on input, 
+    and return details to user
+    """
+    # input
+    data = access_json_data(sys.argv[1])
```

Comment:
Is it a problem that I call data here and in the function below? Should data be in its own function and I call that function in main and get_valid_item_input?